src/lib/components/FieldSelect.svelte

```diff
@@ -144,7 +144,17 @@
 												{option.label}
 											</span>
 											{#if option.badge}
-												<Badge>{option.badge}</Badge>
+												{#if Array.isArray(option.badge)}
+													<div class="field-select-badge">
+														{#each option.badge as badge}
+															<Badge variant={badge === 'OpenAI' ? 'openai' : badge === 'Ollama' ? 'ollama' : undefined}>
+																{badge}
+															</Badge>
```

Comment:
Added icons for Ollama and OpenAI.

![image](https://github.com/user-attachments/assets/10bbe816-b00e-4486-8709-6cd5a1eba2d9)